### PR TITLE
feat: add second functions lab

### LINKED ISSUE #456: chore: add second functions lab to frontend cert (JS only)
Functions are a topic that confuse a lot of campers.
So we should have extra labs for this

The previous workshop and lecture videos will teach campers about functions.

Here is what they will know so far.

- let
- const
- assignment
- reassignment
- basic string usage
- console.log
- camelCase
- string concatenation using the + operator
- template literals
- str.length property
- bracket notation and retrieving characters from strings
- str.indexOf method
- Number()
- Math.floor
- Math.random
- getting a random number from a range
- prompt()
- ===
- if/else if/else
- true and false (truthy and falsey)
- comparison operators(< >=)
- logical operators (|| &&)
- array basics
- push
- pop
- shift
- unshift
- basic object usage
- for loop
- for of loop
- do/while loop
- while loop
- forEach loop
- regular functions
- arrow functions

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by ihechikara**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #456

### CODE REVIEW

File: frontend-cert/js-projects/second-functions/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+1. You should define a function named `askQuestion` with a `question` parameter. The function should return a prompt with `question` as its parameter.
+
+2. You should define a function named `checkAnswer` with two parameters: `userAnswer` and `correctAnswer`. The fucntion should return the comparison of `userAnswer.toLowerCase()` and `correctAnswer.toLowerCase()`.
```
Comment (zairahira) [inline frontend-cert/js-projects/second-functions/user-stories.md:3]:
```suggestion
2. You should define a function named `checkAnswer` with two parameters: `userAnswer` and `correctAnswer`. The function should return the comparison of `userAnswer.toLowerCase()` and `correctAnswer.toLowerCase()`.
```

File: frontend-cert/js-projects/second-functions/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+3. You should define an arrow function named `runQuiz`. Step 4-7 should be created within the `runQuiz` function.
+
+4. You should create an array named `questions`. The array should have objects with `question` and `answer` properties. For example: `{ question: "What is the capital of France?", answer: "Paris" }`, `{ question: "What is 2 + 2?", answer: "4" },`, and so on. Create as many objects as you want.
```
Comment (zairahira) [inline frontend-cert/js-projects/second-functions/user-stories.md:7]: I think here we should put a limit and provide those questions so it is easier to test them.
What do you think about that?

Comment (ihechikara) [reply frontend-cert/js-projects/second-functions/user-stories.md]: I agree. I've updated the user stories 👍

File: frontend-cert/js-projects/second-functions/script.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+function askQuestion(question) {
+    return prompt(question);
+}
+
+function checkAnswer(userAnswer, correctAnswer) {
+    return userAnswer.toLowerCase() === correctAnswer.toLowerCase();
```
Comment (moT01) [inline frontend-cert/js-projects/second-functions/script.js:6]: Is `toLowerCase` covered before this?

File: frontend-cert/js-projects/second-functions/script.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+function askQuestion(question) {
+    return prompt(question);
+}
```
Comment (moT01) [inline frontend-cert/js-projects/second-functions/script.js:3]: I wonder if we should get rid of this - you could just put `prompt(question)` in the `forEach`? That would simplify things a little I think. The user stories would need to be adjusted slightly.

File: frontend-cert/js-projects/second-functions/user-stories.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+1. You should define a function named `askQuestion` with a `question` parameter. The function should return a prompt with `question` as its parameter.
+
+2. You should define a function named `checkAnswer` with two parameters: `userAnswer` and `correctAnswer`. The function should return the comparison of `userAnswer.toLowerCase()` and `correctAnswer.toLowerCase()`.
+
+3. You should define an arrow function named `runQuiz`. Step 4-7 should be created within the `runQuiz` function.
+
+4. You should create an array named `questions`. The array should have objects with `question` and `answer` properties. For example, one object should look like this: `{ question: "What is 2 + 2?", answer: "4" }`. You should create four objects.
```
Comment (moT01) [inline frontend-cert/js-projects/second-functions/user-stories.md:7]:
```suggestion
4. You should create an array named `questions`. The array should have objects with `question` and `answer` properties. For example, one object should look like this: `{ question: "What is 2 + 2?", answer: "4" }`. You should create at least four objects.
```

This gives a little more flexibility in user projects.

**Review by moT01 — COMMENTED** [review]
This is looking good @ihechikara 👍 I left a few suggestions. 

Additionally, I wonder if we should adjust the logs a little bit. When I run the code and enter four correct answers, the logs just say `Correct!` four times. Maybe something to make the logs look like:

```
Question: What is the capital of France?
Correct!

Question: What is 2 + 2?
Wrong. The correct answer is 4
```

That's not really needed, I just think it would be a little better.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**